src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name) {
+        this.name = name;
+        this.quantity = 0;
+    }
+
+    public Fruit(String name, int quantity) {
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+}
```

Comment:
Consider encapsulating the model's fields. It's a common practice to make fields private and provide public getter and setter methods to access these fields, which you have done. However, you should also ensure that the model itself is not mutable if not necessary. If the quantity of the fruit is not meant to change after the object is created, consider removing the setter or making the class immutable.

Reply:
--